packages/@o3r/eslint-plugin/project.json

```
@@ -41,5 +41,9 @@
     "publish": {},
     "documentation": {}
   },
-  "tags": []
+  "tags": [
+    "scope:o3r",
+    "type:workspace-required",
```

why not 
```suggestion
    "workspace-required",
```
I can understand the `type:plugin` but here the package is not exposing a workspace-required 
it's only required for us to have the monorepo working no ?